src/Main.java

```diff
@@ -4,9 +4,59 @@ public class Main {
     public static void main(String[] args) {
         //TIP Press <shortcut actionId="ShowIntentionActions"/> with your caret at the highlighted text
         // to see how IntelliJ IDEA suggests fixing it.
-        System.out.printf("Hello and welcome!");
+        System.out.println("Задача 1");
+        var dog = 8.0;
+        var cat = 3.6;
+        var paper = 763789;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+        System.out.println("Задача 2");
+        dog = dog + 4;
+        cat = cat + 4;
+        paper = paper + 4;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+        System.out.println("Задача 3");
+        dog = dog - 3.5;
+        cat = cat - 1.6;
+        paper = paper - 7639;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+        System.out.println("Задача 4");
+        var friend = 19;
+        friend = (friend + 2)/7;
+        System.out.println(friend);
+        System.out.println("Задача 5");
+        var frog = 3.5;
+        frog = (frog*10)/3.5 + 4;
+        frog = frog*10;
+        System.out.println(frog);
+        frog = frog/3.5;
+        System.out.println(frog);
+        frog = frog + 4;
+        System.out.println (frog);
+        System.out.println(frog);
+        System.out.println("Задача 6");
+        var fighter1 = 78.2;
+        var fighter2 = 82.7;
+        var difference1 =fighter2/fighter1;
+        System.out.println("Разница в весе бйцов " + difference1 + " кг.");
+        System.out.println("Задача 7");
+        var difference2 =fighter2%fighter1;
+        System.out.println("Разница в всесе бойцов " + difference2 + "%");
+        System.out.println("Задача 8");
+        var time = 640;
+        var workingHours = 8;
+        var totalEmployees = time/workingHours;
+        System.out.println("Всего " + totalEmployees + " сотрудников работает в комании!");
+        var newEmployees = totalEmployees + 94;
```

Comment:
можно было так, чтобы не создавать новую переменную
```suggestion
        totalEmployees = totalEmployees + 94;
```